deployment/aws-batch/job-definitions/production-pfb-analysis-run-job.json

```diff
@@ -15,8 +15,16 @@
             "host": {
                 "sourcePath": "/media/nvme0n1/pgdata"
             }
+        }, {
+            "name": "tmp",
+            "host": {
+                "sourcePath": "/media/nvme0n1/tmp"
+            }
         }],
         "environment": [{
+            "name": "NB_MAX_TRIP_DISTANCE",
+            "value": "2680"
```

Comment:
This seems like the wrong place for this.  I think it should be set inside `import_osm.sh`, i.e. right before [the line setting the default for NB_BOUNDARY_BUFFER to equal NB_MAX_TRIP_DISTANCE](https://github.com/azavea/pfb-network-connectivity/blob/develop/src/analysis/import/import_osm.sh#L15) we should have a line setting the default value for NB_MAX_TRIP_DISTANCE (like [in import_neighborhood.sh](https://github.com/azavea/pfb-network-connectivity/blob/develop/src/analysis/import/import_neighborhood.sh#L5))
```
NB_MAX_TRIP_DISTANCE="${NB_MAX_TRIP_DISTANCE:-2680}"
```